lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
+        user1 = create(:user)
+        user2 = create(:user)
+
+        selected_schedule = create(:schedule, program: program)
+        demo_schedule = create(:schedule, program: program)
+        program.update_attributes(selected_schedule: selected_schedule)
+
+        build(:event, program: program, title: 'Demo Event', abstract: 'This is a demo event instance whose state not defined.').save(validate: false)
+        build(:event, program: program, title: 'Demo Rejected Event', state: 'rejected', abstract: 'This is demo event instance in a rejected state.').save(validate: false)
+        build(:event, program: program, title: 'Demo Unconfirmed Event', state: 'unconfirmed', abstract: 'This is a demo event instance in unconfirmed state.').save(validate: false)
+        build(:event, program: program, title: 'Demo Confirmed Event', state: 'confirmed', abstract: 'This is a demo event instance in a confirmed state.').save(validate: false)
+
+        first_scheduled_event = build(:event, program: program, title: 'first_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance.')
+        first_scheduled_event.save(validate: false)
+        second_scheduled_event = build(:event, program: program, title: 'second_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance.')
+        second_scheduled_event.save(validate: false)
+        multiple_speaker_event = build(:event, program: program, title: 'multiple_speaker_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance having multiple speakers.')
+        multiple_speaker_event.save(validate: false)
+
+        create(:event_user, event: multiple_speaker_event, user: user1, event_role: "speaker")
+        create(:event_user, event: multiple_speaker_event, user: user2, event_role: "speaker")
+
+        conference_rooms = conference.venue.rooms
+
+        create(:event_schedule, event: first_scheduled_event, schedule: selected_schedule, start_time: conference.start_date.to_time + 20.hours, room: conference_rooms.first)
```

Comment:
`conference.start_date` would give something like:

`Thu, 02 Mar 2017`

and `conference.start_date.to_time()` would give you:

`2017-03-02 00:00:00 +0100` or `2017-03-01 23:00:00 UTC` (we are using UTC for the times)

To avoid all this confusion I think it is much clear that you write it like that:

`conference.start_date + 9.hours`

this would give you:

`Thu, 02 Mar 2017 09:00:00 UTC +00:00`


Moreover, since https://github.com/openSUSE/osem/pull/1143 was merged you also need to take into account the `start_hour`. So you need to write something like:

`conference.start_date + conference.start.hour.hours`

or if you want for example to start one hour later than the conference start time:

`conference.start_date + conference.start.hour.hours + 1.hours`


You could also use that you now that for your conference the `start_hour` is 9, but I think it is more readable if you write the `start_hour` so you don't have to find out what that 9 is when reading this.